pkg/deploy/api/types.go

```diff
@@ -257,6 +257,8 @@ type DeploymentConfig struct {
 	// Details are the reasons for the update to this deployment config.
 	// This could be based on a change made by the user or caused by an automatic trigger
 	Details *DeploymentDetails
+	// Replicas is the total number of pods targeted by this deploymentConfig.
+	Replicas int
```

Comment:
This name could get confusion -- we now have `DeploymentConfig.Replicas` and `DeploymentConfig.Template.Replicas`, one of which is read-only, and one of which now scales the DeploymentConfig, and it's not particularly clear which is which based on the name.  Since we don't have a status object here, perhaps this should be `TotalReplicas`, `CurrentReplicas`, or something of the sort, just to avoid confusion.